test/tests/functional/pbs_hook_execjob_end.py

```diff
@@ -457,3 +457,21 @@ def test_qrun_arrayjob_on_mom_breakdown(self):
         self.server.expect(JOB, {'job_state': 'B'}, jid)
         subjid_1 = j.create_subjob_id(jid, 1)
         self.common_steps(subjid_1, self.mom)
+
+    def test_mom_restart(self):
+        """
+        Test to restart mom while execjob_end hook is running
+        """
+        hook_name = "end_hook"
+        attr = {'event': 'execjob_end', 'enabled': 'True', 'alarm': '100'}
+        self.server.create_import_hook(hook_name, attr, self.hook_body)
```

Comment:
Also, I think you will need long-running execjob_end hook, current hook only sleeps for 5 sec which is too less

Reply:
Its actually 10s. I'm using the hook_body which is defined in the setUp(). This is good enough right?

Reply:
you set the hook alarm time to 100, means you want long-running hook right? so, I don't think 10s is enough.

Reply:
I have changed the values.